src/res/flame.html

```diff
@@ -3,33 +3,204 @@
 <head>
 <meta charset='utf-8'>
 <style>
-	body {margin: 0; padding: 10px 10px 22px 10px; background-color: #ffffff}
-	h1 {margin: 5px 0 0 0; font-size: 18px; font-weight: normal; text-align: center}
-	header {margin: -24px 0 5px 0; line-height: 24px}
-	button {font: 12px sans-serif; cursor: pointer}
-	p {position: fixed; bottom: 0; margin: 0; padding: 2px 3px 2px 3px; outline: 1px solid #ffc000; display: none; overflow: hidden; white-space: nowrap; background-color: #ffffe0}
+	body {margin: 0; padding: 5px 5px 15px 5px; background-color: #ffffff}
+	h1 {margin: 0; font-size: 18px; font-weight: normal; flex-grow: 1; text-align: center}
+	header {margin: 0; line-height: 20px; text-align: right; font-size: 11px; color: #666;}
+	.title-toolbar-container {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		margin-bottom: 2px;
+		background-color: #ffffff;
+		padding: 1px 2px;
+		border: none;
+		border-radius: 0;
+	}
+	.title-section {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		flex-grow: 1;
+	}
+	.toolbar-section {
+		display: flex;
+		align-items: center;
+		gap: 2px;
+	}
+	.left-toolbar {
+		display: flex;
+		align-items: center;
+		gap: 2px;
+		width: 150px;
+	}
+	.right-section {
+		display: flex;
+		align-items: center;
+		width: 150px;
+	}
+	p {position: fixed; bottom: 0; margin: 0; padding: 1px 2px 1px 2px; outline: 1px solid #ffc000; display: none; overflow: hidden; white-space: nowrap; background-color: #ffffe0}
 	a {color: #0366d6}
 	#hl {position: absolute; display: none; overflow: hidden; white-space: nowrap; pointer-events: none; background-color: #ffffe0; outline: 1px solid #ffc000; height: 15px}
-	#hl span {padding: 0 3px 0 3px}
+	#hl span {padding: 0 2px 0 2px}
```

Comment:
Why are all these changes in margins and paddings?
Now, frame title jumps whenever I hover a pointer over it.

Reply:
Reverting these paddings, doesn't fix the jumping. I can see that behavior on master as well:

![jumping-frame-title](https://github.com/user-attachments/assets/e9f3aae9-c319-4b82-8bc9-a63b99f6de21)

Reply:
Hm.. might depend on the OS/browser. It does not jump in Chrome.